docs/website/references/cli/defradb_identity_new.md

```diff
@@ -0,0 +1,45 @@
+## defradb identity new
+
+Generate a new identity
+
+### Synopsis
+
+Generate a new identity
```

Comment:
suggestion: related to previous comment: would be nice to have some more details.

Reply:
Since this is auto gen documentation, will keep it to that one thread.